utils.go

```diff
@@ -0,0 +1,29 @@
+package firecracker
+
+import (
+	"context"
+	"time"
+)
+
+const (
+	defaultAliveVMMCheckDur = 10 * time.Millisecond
+)
+
+// waitForAliveVMM will check for periodically to see if the firecracker VMM is
+// alive. If the VMM takes too long in starting, an error signifying that will
+// be returned.
+func waitForAliveVMM(ctx context.Context, client *Client) error {
+	t := time.NewTicker(defaultAliveVMMCheckDur)
+	defer t.Stop()
+
+	for {
+		select {
+		case <-ctx.Done():
+			return ctx.Err()
+		case <-t.C:
+			if _, err := client.GetMachineConfig(); err == nil {
```

Comment:
This is almost same as `waitForSocket`. Would it make sense to refactor/leave one waiter?

Reply:
Yea, issue is the `exitchan` handles early exit of the machine which isn't needed for the utility. The code is small and won't ever change, so I am not too worried about it.